packages/experience/src/components/IdentifierRegisterForm/index.tsx

```diff
@@ -87,7 +87,7 @@ const IdentifierRegisterForm = ({ className, autoFocus, signUpMethods }: Props)
           return;
         }
 
-        await onSubmit(type, value, captchaToken);
+        await onSubmit(type, value, await executeCaptcha());
```

Comment:
I was considering extracting the interaction initialization API with the captcha context into a react hook and returning a unified initInteractionAPI. This way, all submission touchpoints wouldn’t need to handle the captcha separately. Not a blocker,  can create an improvement issue.

Reply:
LOG-11173